runtime/compiler/aarch64/codegen/J9TreeEvaluator.hpp

```diff
@@ -48,6 +48,8 @@ class OMR_EXTENSIBLE TreeEvaluator: public J9::TreeEvaluator
    {
    public:
 
+   TR::Register * flushEvaluator(TR::Node *node, TR::CodeGenerator *cg);
```

Comment:
Will this function be added to the evaluator table for the fence opcodes?

Reply:
My model for this was PPC's version. Right now I think they are modifying/initializing the function pointers of the original instructions to point to `flushEvaluator`:
```
extern void TEMPORARY_initJ9PPCTreeEvaluatorTable(TR::CodeGenerator *cg)
   {
   TR_TreeEvaluatorFunctionPointer *tet = cg->getTreeEvaluatorTable();

   (...)
   tet[TR::allocationFence] = TR::TreeEvaluator::flushEvaluator;
   tet[TR::loadFence] = TR::TreeEvaluator::flushEvaluator;
   tet[TR::storeFence] = TR::TreeEvaluator::flushEvaluator;
   tet[TR::fullFence] = TR::TreeEvaluator::flushEvaluator;
   (...)
   }
```
Is this the same as what you meant? 
Is there a preferred method of calling a function like the one I'm working on? (For example, creating a helper and just calling it from different evaluators like we've seen in OMR)

Reply:
yes, that's what I meant, but only for AArch64 obviously :-) .  Otherwise, the function will never be called.